redaxo/src/core/lib/sql/sql.php

```diff
@@ -743,7 +743,21 @@ public function getValue($column)
      */
     public function getArrayValue($column)
     {
-        return json_decode($this->getValue($column), true);
+        $value = $this->getValue($column);
+        if (null === $value) {
+            return [];
+        }
+
+        try {
+            $decoded = @json_decode($value, true, 512, JSON_THROW_ON_ERROR);
```

Comment:
```suggestion
            $decoded = json_decode($value, true, 512, JSON_THROW_ON_ERROR);
```

Fehlerunterdrückung ist dann denke ich überflüssig.

Reply:
confirmed, gibt keine warning:

https://3v4l.org/vTV99